docs/source/en/api/pipelines/fabric.md

```diff
@@ -0,0 +1,31 @@
+## changes required
```

Comment:
Guessing this will undergo changes?

Reply:
I forgot to remove the comment, It's done.

Should I make it something similar to - https://github.com/huggingface/diffusers/blob/main/docs/source/en/api/pipelines/stable_diffusion/stable_diffusion_xl.md

Reply:
Yup. We follow a consistent documentation structure. So, feel free to pick a pipeline doc and adapt.